eo-phi-normalizer/src/Language/EO/Phi/Rules/Common.hs

```diff
@@ -146,8 +146,8 @@ nuCount obj = count isNu (objectBindings obj) + sum (map (sum . map nuCount . va
   values (AlphaBinding _ obj') = [obj']
   values _ = []
 
-intToBytesObject :: Int -> Object
-intToBytesObject n = Formation [DeltaBinding $ Bytes $ insertDashes $ pad $ showHex n ""]
+intToBytes :: Int -> Bytes
+intToBytes n = Bytes $ insertDashes $ pad $ showHex n ""
```

Comment:
This does not work for negative integers.

Reply:
Does it need to? It's only used with nonnegative numbers

Reply:
Ok, just realized that an `arbitrary` integer can be negative. Will fix that